rele/config.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, setting):
         )
         self._encoder_path = setting.get("ENCODER_PATH", DEFAULT_ENCODER_PATH)
         self.publisher_timeout = setting.get("PUBLISHER_TIMEOUT", 3.0)
-        self.threads_per_subscription = setting.get("THREADS_PER_SUBSCRIPTION", 10)
+        self.threads_per_subscription = setting.get("THREADS_PER_SUBSCRIPTION", 3)
```

Comment:
3? Are you a rebel against the age-old computer science preference for multiples of 2? :smile:

Reply:
Haha No Im not a rebel in this regard. If we prefer 2, I can change it to 2.